packages/plugin-core/src/telemetry/web/WebTelemetryClient.ts

```diff
@@ -0,0 +1,99 @@
+import { AppNames } from "@dendronhq/common-all";
+import axios, { AxiosRequestConfig } from "axios";
+import { inject, injectable } from "tsyringe";
+import * as vscode from "vscode";
+import { ITelemetryClient } from "../common/ITelemetryClient";
+
+/**
+ * This implementation talks to Segment services via their HTTP API. It's safe
+ * to use in both web and node contexts.
+ *
+ * Note: the Segment Javascript library was not used here because it requires a
+ * browser 'window' object, which is not available to web extensions.
+ */
+@injectable()
+export class WebTelemetryClient implements ITelemetryClient {
+  constructor(
+    @inject("anonymousId") private anonymousId: string,
+    @inject("extVersion") private extVersion: string
+  ) {}
+
+  /**
+   * This key talks to the 'Dendron-Web-Extension' source in Segment. NOTE: this
+   * is different from the 'ide-prod' source.
+   */
+  private DENDRON_WEB_EXTENSION_SEGMENT_WRITE_KEY =
```

Comment:
can we obfuscate this like we do currently with the regular segment key?

Reply:
How is this obfuscated? Aren't they just here in plaintext? https://github.com/dendronhq/dendron/blob/766ba6db92e5b17cd6e9d24b478e5249e9acf3c4/packages/common-all/src/config.ts#L24